### ISSUE TITLE
Document Details: Unable to click on outline when popover has focus

### ISSUE THREAD

**Issue by shaunandrews** [WordPress/gutenberg#26896]
![document details popover](https://user-images.githubusercontent.com/191598/98830444-6a7f5480-2408-11eb-9e74-13604a37f02b.gif)

The document details popover displays both stats (like word count) and the document outline. The stats are _not_ interactive, but the outline _is_ — however this feels broken.

When you first open the popover, the keyboard focus is on the popover itself — notice how the popover gets the blue border. This breaks the interaction with the document outline; I have to move keyboard focus _first_, and then I can click on the items in the outline list.
